Makefile

```diff
@@ -12,14 +12,27 @@ IMAGE_TAG ?= latest
 OPERATOR_IMAGE ?= cluster-network-addons-operator
 REGISTRY_IMAGE ?= cluster-network-addons-registry
 
```

Comment:
Let's track source code with Makefile so we can make targets depend on it
cmd_sources := $(wildcard ./cmd/**/*.go)
pkg_sources := $(wildcard ./pkg/**/*.go)
apis_sources := $(wildcard ./pkg/**/apis/**/*.go

Reply:
Done